lighthouse/validators/samples_declarations.py

```diff
@@ -88,3 +90,48 @@ def pre_samples_declarations_post_callback(request):
 
     return request
 
+
+def build_clean_elems_object(items, request):
+    i = 0
+    clean_elems = {}
+    while i < len(items):
+        if items[i]["_status"] == "OK":
+            clean_elems[i] = request.json[i]
+        i = i + 1
+    return clean_elems
```

Comment:
huh, didn't realise python doesn't have a `for i=0 i<10 i++` style loop

Reply:
looks like you can achieve a similar thing with `for x in range(6):  print(x)` though...
anyway, just commenting, the while loop is fine :)

Reply:
yes, much better with range, changing